trusted/crypto/sys/types/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Rust FFI types for the SGX SDK trusted crypto library (tcrypto).
```

Comment:
How about we make this `*-sys*`  crates `no_std`?

Reply:
that's a good point, the trusted side will need to be no_std since it will be in the enclave.
will work on updating

Reply:
added the `no_std`